src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private final double side;
+
+    public Square(double side, String color) {
+        super(color);
+        this.side = side;
+    }
+
+    @Override
+    public double calculateArea() {
+        return side * side;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: square, area: " + calculateArea() + " sq"
```

Comment:
The method draw() should not have a direct call to System.out.println(). This breaks the single responsibility principle. Consider passing a PrintStream in the constructor of this class and use it to print the message.